lib/app_manifest.rb

```diff
@@ -36,9 +36,9 @@ def canonicalize_env(manifest)
       canonicalize_key(manifest, :env) do |env|
         Hash[
           env.map do |key, value|
-            if value.is_a? String
+            if !value.is_a?(Hash)
               value = {
-                value: value,
+                value: value.to_s,
```

Comment:
@dmathieu another case of a user trying to use a non string/hash value on envs. https://support.heroku.com/tickets/461969?ts=1490087649. Did you guys plan working on that?